### PR TITLE
 update changelog with python 2.7 support removed 

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
can you also yank the following releases, and create .patch0 versions with the requires_python line in

0.36.8 (2020-07-08)
0.36.7 (2020-07-02)
0.36.6 (2020-05-24)

this allows pip on python 2.7 to install the correct version

File: setup.py [hunk 1]
```diff
@@ -48,7 +47,7 @@ def get_version():
     'phone': ['phonenumbers>=5.9.2'],
     'password': ['passlib >= 1.6, < 2.0'],
     'color': ['colour>=0.0.4'],
-    'ipaddress': ['ipaddr'] if not PY3 else [],
+    'ipaddress': [],
     'enum': ['enum34'] if sys.version_info < (3, 4) else [],
```
Comment (graingert) [inline setup.py:51]:
```suggestion
    'enum': [],
```

**Comment by graingert** [comment]
0.37.0
0.37.1

will also need yanking @kvesteri

**Comment by kvesteri** [comment]
Closed via https://github.com/kvesteri/sqlalchemy-utils/pull/519

**Comment by graingert** [comment]
@kvesteri it looks like you missed a commit when cherry-picking my PR (and my attribution)

**Comment by kurtmckee** [comment]
@kvesteri Updating the changelog to note when Python 2.7 support was dropped seems worthwhile, even so long after the fact.

I recommend merging this PR and then fixing the capitalization of the changelog line to match the rest of the changelog.

**Comment by kvesteri** [comment]
Fixed in https://github.com/kvesteri/sqlalchemy-utils/commit/3ba25a6177d4b64698ba23f19ecb48ace5944ed1

**Pull request closed without merging**